qiskit/utils/optionals.py

```diff
@@ -99,9 +99,9 @@
         format.  There are converter methods on :class:`.DAGCircuit` if NetworkX is present.
 
     * - .. py:data:: HAS_NLOPT
-      - `NLOpt <https://nlopt.readthedocs.io/en/latest/>`__ is a nonlinear optimisation library,
-        used by the global optimizers in :mod:`.algorithms.optimizers`.  See installation details in
-        :ref:`installing-nlopt`.
+      - `NLopt <https://nlopt.readthedocs.io/en/latest/>`__ is a nonlinear optimization library,
+        used by the global optimizers in the :mod:`.algorithms.optimizers` module. It must
+        be installed in order to use them.
```

Comment:
The extra sentence and the end is a bit superfluous - the entire documentation of this module is about packages that need to be installed to use extra functionality.  We can probably just cut it.

(Thanks for fixing my poor US localisation as well.)